src/app.js

```diff
@@ -1 +1,46 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios, {defaults} from 'axios'
+
+async function getCountryList() {
+    try {
+        const result = await axios.get('https://restcountries.com/v2/all?fields=name,flag,population,region')
+        let list = result.data
+        list.sort((a, b) => {
+            return a.population - b.population
+        })
+        return list
+    } catch (error) {
+        console.error(error)
+    }
+}
+
+async function htmlList() {
+    const list = await getCountryList()
+    let cList = document.getElementById("country-list")
+    list.map((countryList) => {
```

Comment:
Hier had je nog spreading kunnen toepassen, zodat je niet overal `countryList.` vóór hoeft te zetten:
```suggestion
    list.map(({ name, flag, region, population }) => {
```